modules/compat-libraries-32bit.yml

```
@@ -0,0 +1,22 @@
+x-compat-i386-opts: &compat_i386_opts
+  prepend-pkg-config-path: /app/lib32/pkgconfig:/usr/lib/i386-linux-gnu/pkgconfig
+  ldflags: -L/app/lib32
+  prepend-path: /usr/lib/sdk/toolchain-i386/bin
+  env:
+    CC: i686-unknown-linux-gnu-gcc
+    CXX: i686-unknown-linu-gnu-gcc
+    libdir: /app/lib32
+
+name: compat-libraries-32bit
+buildsystem: simple
+commands: []
+modules:
+  - name: google-perfools
+    buildsystem: autotools
+    build-options:
+      arch:
+        x86_64: *compat_i386_opts
+    sources:
+      - type: git
+        url: https://github.com/gperftools/gperftools.git
+        commit: perftools-1.6-0-gfd52267cb91a42f839c7be6a348e014cfdbcaca9
```

Seems like gperftools 1.6 won't build with newer glibc: gperftools/gperftools#350
```suggestion
        commit: 0b87a55895a184bcc529b4ce13c46c0411182b85
        tag: google-perftools-1.8.3
```

---

Is this just head or some specific commit?

---

It's the commit tagged as version 1.8.3.